tests/test_ha.py

```diff
@@ -431,6 +431,7 @@ def test_follow_in_pause(self):
         self.assertEqual(self.ha.run_cycle(), 'PAUSE: no action. I am (postgresql0)')
 
     @patch.object(Rewind, 'rewind_or_reinitialize_needed_and_possible', Mock(return_value=True))
+    @patch.object(Rewind, 'check_leader_has_run_checkpoint', Mock(return_value=True))
```

Comment:
Please revert changes in the test_ha.py.
With the latest changes in rewind.py they are not necessary.

Reply:
done so now